pkg/validation/app.go

```diff
@@ -16,14 +16,15 @@ import (
 )
 
 const (
-	catalogNotFoundTemplate         = "catalog %#q not found"
-	nameTooLongTemplate             = "name %#q is %d chars and exceeds max length of %d chars"
-	nameNotFoundReasonTemplate      = "name is not specified for %s"
-	targetNamespaceNotAllowed       = "target namespace %s is not allowed for in-cluster apps"
-	namespaceNotFoundReasonTemplate = "namespace is not specified for %s %#q"
-	labelInvalidValueTemplate       = "label %#q has invalid value %#q"
-	labelNotFoundTemplate           = "label %#q not found"
-	resourceNotFoundTemplate        = "%s %#q in namespace %#q not found"
+	catalogNotFoundTemplate           = "catalog %#q not found"
+	nameTooLongTemplate               = "name %#q is %d chars and exceeds max length of %d chars"
+	nameNotFoundReasonTemplate        = "name is not specified for %s"
+	targetNamespaceNotAllowedTemplate = "target namespace %s is not allowed for in-cluster apps"
```

Comment:
"Fixed" this, the `Template` part was missing making it inconsistent with the rest of constants